src/sentry/api/endpoints/organization_details.py

```diff
@@ -326,7 +326,17 @@ def save(self, *args, **kwargs):
         return super(OwnerOrganizationSerializer, self).save(*args, **kwargs)
 
 
+class OrganizationDetailsPermission(OrganizationPermission):
+    scope_map = {
+        "GET": OrganizationPermission.scope_map["GET"] + ["org:billing"],
```

Comment:
@wmak  `["org:billing"]` shouldn't be here.

Reply:
@wmak generally the strategy is to be able to control this scope from `getsentry` side.

Reply:
@dashed I added it here since we need this endpoint to load the settings page.